esp-hal/MIGRATING-1.0.0-beta.1.md

```diff
@@ -67,3 +67,8 @@ Additionally, the `configure` methods have been renamed to `configure_tx` and
 -};
 + let rx_channel = rmt.channel2.configure_rx(pin, rx_config);
 ```
+
+## The RMT error type has been marked `non_exhaustive`
```

Comment:
This is fine as a changelog entry, I don't think we need a migration guide paragraph about it.

Reply:
Done :+1: